### PR TITLE
Allow latex input in "latex" fields

### PR DESCRIPTION
**Pull request by bixel**

### CODE REVIEW

File: database/events/json_forms.py [hunk 1]
```diff
@@ -38,6 +40,10 @@ def create_wtf_field(name, schema, required=True):
     if schema['type'] == 'boolean':
         return wtforms.BooleanField(**kwargs)
 
+    if schema['type'] == 'latex':
```
Comment (MaxNoe) [inline database/events/json_forms.py]: shouldn't this by `{'type': 'string', 'format': 'latex'}? Jsonschema has no type `latex`

Comment (MaxNoe) [reply database/events/json_forms.py]: See: https://json-schema.org/understanding-json-schema/reference/string.html

There are several predefined formats, e.g. email or date.

Comment (MaxNoe) [reply database/events/json_forms.py]: See e.g. https://github.com/pep-dortmund/member-database/pull/38

Comment (bixel) [reply database/events/json_forms.py]: Ok thanks yes that makes sense.

Comment (MaxNoe) [reply database/events/json_forms.py]: Just made the fix

**Comment by MaxNoe** [comment]
This is pretty awesome, thx!

File: database/events/json_forms.py (hunk 1 above)
Comment (bixel) [reply database/events/json_forms.py]: Cool! Sorry for being slow. Can't spent as much time as I would like to right now 😁

**Pull request merged**